src/java.desktop/macosx/classes/sun/lwawt/macosx/CPlatformWindow.java

```diff
@@ -1002,11 +1002,13 @@ public void setWindowState(int windowState) {
 
         int prevWindowState = peer.getState();
         if (prevWindowState == windowState) return;
+        System.out.println("check1");
```

Comment:
Do you need this debug statement ( all check1-check8  ) ?

Reply:
oops, forgot about those